### PR TITLE
thinkity thonk donk

### PR DESCRIPTION
**Pull request by Kyousei**

### CODE REVIEW

File: DiscordLibrary.py [hunk 1]
```diff
@@ -150,7 +150,7 @@ def check(sql):
 
 		sql = "SELECT GuildID FROM Guilds WHERE GuildID = '" + str(guild.id) + "'"
 		check(sql)
-		if sql == False:
+		if guild.id is not in data[0]:
```
Comment (Kyousei) [inline DiscordLibrary.py]: This will check if the guild ID is in the all data the bot gets from the database.

Comment (Kyousei) [reply DiscordLibrary.py]: If it isn't, you'll most likely going to get IndexError

Comment (Kyousei) [reply DiscordLibrary.py]: By the way, hello people who're reading #bot-development. :^)

Comment (Kyousei) [reply DiscordLibrary.py]: Reply to me in <#386954275185754112>

**Comment by Deivedux** [comment]
Are you sure this is the way it should work?
```
    if guild.id is not in data[0]:
                        ^
SyntaxError: invalid syntax
```

**Comment by Kyousei** [comment]
Damn. xd

**Comment by Kyousei** [comment]
Try that.

**Comment by Deivedux** [comment]
Like you said, I got an IndexError. 🤔 So I'm guessing I should use `try except` to bypass that? Since `if else` doesn't seem like it's doing it's job properly.

**Comment by Kyousei** [comment]
You got IndexError because it wasn't found.
Put a try: and except: to bypass it. 😅

**Comment by Deivedux** [comment]
Alright thanks for this at least, I'll figure out the rest (if I know what I'm doing).

**Comment by Kyousei** [comment]
:^)

**Pull request merged**